tests/pinkopy/test_helper.py

```diff
@@ -0,0 +1,49 @@
+import uuid
+
+import requests_mock
+
+
+def mock_session(base_session, service='http://' + str(uuid.uuid4()),
+                 user=str(uuid.uuid4()), pw=str(uuid.uuid4()), token=str(uuid.uuid4()),
+                 content_type='application/json', clients=str(uuid.uuid4())
+                 ):
+    get_response = {
+        'App_GetClientPropertiesResponse': {
+            'clientProperties': clients
+        }
+    }
+    post_payload = {
+        'DM2ContentIndexing_CheckCredentialResp': {
+            '@token': token
+        }
+    }
+    headers = {
+        'Accept': content_type,
+        'Content-type': content_type
+    }
+    with requests_mock.mock() as m:
+        m.get(service + '/Client', json=get_response)
```

Comment:
I think what is confusing here is the way they construct the call. In requests, these parameters represent the requests, but in the mock it seems they represent the response. Neat.